src/main/java/lotto/model/LotteryMachine.java

```diff
@@ -0,0 +1,42 @@
+package lotto.model;
+
+import java.util.EnumMap;
+import java.util.List;
+import java.util.Map;
+import lotto.model.enums.WinningMoney;
+
+public class LotteryMachine {
+
+    private Map<WinningMoney, Integer> store;
+
+    private PersonLotto personLotto;
+
+    private WinningLotto winningLotto;
+
+    public LotteryMachine(PersonLotto personLotto, WinningLotto winningLotto) {
+        this.store = new EnumMap<>(WinningMoney.class);
+        this.personLotto = personLotto;
+        this.winningLotto = winningLotto;
+    }
+
+    public Map<WinningMoney, Integer> drawingLotto(Bonus bonus) {
+        personLotto.getPurchaseLotto().forEach(
+                lotto -> {
+                    int matchCount = checkMatch(lotto);
+                    WinningMoney result = WinningMoney.from(matchCount, checkBonus(lotto, bonus));
+                    store.put(result, store.getOrDefault(result, 0) + 1);
+                }
+        );
```

Comment:
추가로, Map<> 자료구조에 바로 `put()` 메서드를 사용하는 대신, 해당 자료구조를 일급 컬렉션으로 작성하여 람다식의 바디에 해당하는 부분을 메서드로 따로 구현할 수 있을 것 같아요. 그럼 람다식도 메서드 레퍼런스를 통해 훨씬 간결하게 줄일 수 있습니다.
예를 들어 `Store` 일급 컬렉션 내에 `addLotto()` 메서드가 있다면 다음과 같이 사용할 수 있을거에요.
```suggestion
        personLotto.getPurchaseLotto()
                .forEach(store::addLotto);
```